packages/unlock-js/src/__tests__/integration/single.js

```diff
@@ -4,26 +4,26 @@
  *
  * Usage:
  *
- * yarn hardhat test:integration src/__tests__/integration/lock/purchaseKey.js \
- *     --lock-version 9 \
- *     --unlock-version 10 \
+ * 1. mv single.js single.test.js
+ * 2. edit `testPath` value
+ * 3. yarn vitest
  * */
 import locks from '../helpers/fixtures/locks'
 import nodeSetup from '../setup/prepare-eth-node-for-unlock'
+import { describe, it, expect, beforeAll } from 'vitest'
+
 import {
   chainId,
   setupTest,
   setupLock,
   versionEqualOrAbove,
 } from '../helpers/integration'
 
-// Increasing timeouts
-jest.setTimeout(3000000)
-
-const unlockVersion = process.env.UNLOCK_JS_JEST_RUN_UNLOCK_VERSION || 'v11'
+const unlockVersion = process.env.UNLOCK_JS_TEST_RUN_UNLOCK_VERSION || 'v11'
 const publicLockVersion =
-  process.env.UNLOCK_JS_JEST_RUN_PUBLIC_LOCK_VERSION || 'v11'
-const testPath = process.env.UNLOCK_JS_JEST_RUN_TEST_PATH
+  process.env.UNLOCK_JS_TEST_RUN_PUBLIC_LOCK_VERSION || 'v11'
```

Comment:
same here

Reply:
yes. Also this script still useful in dev but does not run in CI. The use in dev is tedious as it requires to be renamed to `single.test.js` to be run...